src/config.rs

```diff
@@ -230,13 +230,17 @@ impl Cfg {
             .transpose()?;
 
         let dist_root_server = match process().var("RUSTUP_DIST_SERVER") {
-            Ok(ref s) if !s.is_empty() => s.clone(),
+            Ok(s) if !s.is_empty() => {
+                info!("`RUSTUP_DIST_SERVER` has been set to `{s}`");
```

Comment:
Using `info!()` will interfere with `exact` tests. We need to introduce `...` to those tests and make it skip the first line(s), like we did in the UI tests.

Reply:
Maybe it's better to debug-print those stuff...

Reply:
What if you log at debug! not info!.

Reply:
@rbtcollins Got a bit sidetracked due to personal reasons. Will try.

Reply:
@rbtcollins Done, although the visibility of this feature has thus gone down a bit. I think a docs change should compensate.

Reply:
@rbtcollins Done.